### PR TITLE
ui: updated ux for database service name field in database ingestion form

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes :

updated UX for database service name field in database ingestion form

#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :
image
image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

**Review by Sachin-chaurasiya — DISMISSED** [review]

File: openmetadata-ui/src/main/resources/ui/src/components/AddIngestion/Steps/ConfigureIngestion.tsx [hunk 1]
```diff
@@ -490,14 +484,17 @@ const ConfigureIngestion = ({
...
-          name="name"
-          type="text"
+          mode="tags"
+          placeholder={t('label.add-entity', {
+            entity: t('label.database-service-name'),
+          })}
+          style={{ width: '100%' }}
           value={databaseServiceNames}
-          onChange={(e) => handleDashBoardServiceNames(e.target.value)}
+          onChange={(value) => handleDashBoardServiceNames(value)}
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/AddIngestion/Steps/ConfigureIngestion.tsx:497]:
```suggestion
          onChange={handleDashBoardServiceNames}
```

File: openmetadata-ui/src/main/resources/ui/src/components/AddIngestion/Steps/ConfigureIngestion.tsx [hunk 2]
```diff
@@ -490,14 +484,17 @@ const ConfigureIngestion = ({
...
         </p>
-        <input
-          className="tw-form-inputs tw-form-inputs-padding"
+        <Select
+          allowClear
           data-testid="name"
           id="name"
-          name="name"
-          type="text"
+          mode="tags"
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/AddIngestion/Steps/ConfigureIngestion.tsx:491]: Can we also add some instruction to let them know that they can add multiple

Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/components/AddIngestion/Steps/ConfigureIngestion.tsx:491]: updated description message

File: openmetadata-ui/src/main/resources/ui/src/components/AddIngestion/Steps/ConfigureIngestion.tsx (hunk 1 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/components/AddIngestion/Steps/ConfigureIngestion.tsx]: done

**Review by chirag-madlani — APPROVED** [review]

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**